storage/s3-sdk/src/main/java/ListGcsBuckets.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * """This sample shows how to list Google Cloud Storage (GCS) buckets using the AWS S3 SDK with the
```

Comment:
Is this comment necessary? Seems like stating it in the README is probably best, or else trimming it down and including it inside the snippet.